### PR TITLE
Fix cloudwatch alarm failure to create (#56)

### LINKED ISSUE #56: Cloudwatch alarms insufficient data
I've created a redis without enabling the automatic_failover, only one node. The problem is that alarms at cloudwatch shows insufficient data.
The configured dimension value for CacheClusterId in the alarm is "myredis" and if I see in cloudwatch the current metrics having data are in dimension value "myredis-001".

### PR DESCRIPTION
**Pull request by varunpalekar**
## what
* Create multiple cloudwatch alarms for each node of cluster.

## why
* You can't create a cloudwatch alarm on cluster, need to create on each node of cluster.

## references
* closes #56

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -117,8 +118,8 @@ resource "aws_elasticache_replication_group" "default" {
 # CloudWatch Resources
 #
 resource "aws_cloudwatch_metric_alarm" "cache_cpu" {
-  count               = var.enabled ? 1 : 0
-  alarm_name          = "${module.label.id}-cpu-utilization"
+  count               = var.enabled ? lenght(local.elasticache_member_clusters) : 0
```
Comment (vanveele) [inline main.tf:121]:
```suggestion
  count               = var.enabled ? length(local.elasticache_member_clusters) : 0
```

Comment (varunpalekar) [reply main.tf]: Thanks for the review, updated the code for same,

**Comment by joe-niland** [comment]
@varunpalekar please have a look at this issue causing the tests to fail:

image

**Comment by varunpalekar** [comment]
@joe-niland Thanks for the review. I updated the code respectively.
I also find an issue when running on a new infra, also fixed that.

**Review by joe-niland — APPROVED** [review]

**Comment by joe-niland** [comment]
Thanks for your contribution @varunpalekar. Released as https://github.com/cloudposse/terraform-aws-elasticache-redis/releases/tag/0.23.0

**Pull request merged**